### PR TITLE
Doc change to add reboot compute host section in operations.

### PR DESCRIPTION
**Pull request by rajdharwadkar**

### CODE REVIEW

File: doc/source/operations.rst [hunk 1]
```diff
@@ -234,6 +234,23 @@ following information to the "osh" section under "charts":
    such as Deckhand and Shipyard belong under "ucp", OpenStack services belong
    under "osh", and infrastructure components belong under "osh_infra".
 
+Reboot Compute Host
+===================
+
+Before reboot compute host, Shutdown all VM(s) from that compute host.
```
Comment (jgu17) [inline doc/source/operations.rst]: Shutdown->shutdown? lower case 's'

Comment (jgu17) [inline doc/source/operations.rst]: all Nova VM(s)

File: doc/source/operations.rst [hunk 2]
```diff
@@ -234,6 +234,23 @@ following information to the "osh" section under "charts":
...
+
+After reboot the compute host, It is possible that the pods when started come up out of order.
+If this happens, you might see symptoms of the vm not getting an ip address.
```
Comment (jgu17) [inline doc/source/operations.rst]: the vm->the Nova VM(s)

File: doc/source/operations.rst [hunk 3]
```diff
@@ -234,6 +234,23 @@ following information to the "osh" section under "charts":
...
+To address this problem, run the following commands
```
Comment (jgu17) [inline doc/source/operations.rst]: add ':' at the end?

File: doc/source/operations.rst [hunk 4]
```diff
@@ -234,6 +234,23 @@ following information to the "osh" section under "charts":
...
+
+.. code-block:: console
+
+   kubectl get pods -o wide | grep ovs-agent | grep <compute name>"
+   kubectl delete pod -n openstack <ovs-agent pod name>
+
+This should restart the ovs-agent pod and reconfigure the vxlan tunnel network configuration.
```
Comment (jgu17) [inline doc/source/operations.rst]: @sgrasley @gracetai5 did we have to stop/start the Nova servers?

Comment (jgu17) [inline doc/source/operations.rst]: One nit... restart Neutron OVS agent pod.

Comment (sgrasley) [reply doc/source/operations.rst]: We did not need to as part of troubleshooting I was involved in.

File: doc/source/operations.rst [hunk 5]
```diff
@@ -234,6 +234,23 @@ following information to the "osh" section under "charts":
    such as Deckhand and Shipyard belong under "ucp", OpenStack services belong
    under "osh", and infrastructure components belong under "osh_infra".
 
+Reboot Compute Host
+===================
+
+Before reboot compute host, shutdown all Nova VM(s) from that compute host.
+
+After reboot the compute host, It is possible that the pods when started come up out of order.
```
Comment (jgu17) [inline doc/source/operations.rst]: one more nit... It->it

File: doc/source/operations.rst [hunk 6]
```diff
@@ -234,6 +234,23 @@ following information to the "osh" section under "charts":
...
+
+Before reboot compute host, shutdown all Nova VM(s) from that compute host.
+
+After reboot the compute host, it is possible that the pods when started come up out of order.
+If this happens, you might see symptoms of the Nova VM(s) not getting an ip address.
+To address this problem, run the following commands:
+
+.. code-block:: console
+
+   kubectl get pods -o wide | grep ovs-agent | grep <compute name>"
```
Comment (sgrasley) [inline doc/source/operations.rst]: My nit extra "

**Pull request merged**